### DISCUSSION TITLE
Questions about web analytics in docusaurus

### DISCUSSION THREAD

**Discussion by KyrieCui** [vercel/community#2150] [Help]
### Summary

Hello, Sir or Madam,
I have deployed a web with docusaurus-2. If I want to use web analytics I have to add some code about @vercel/analytics in my project. I want to ask which file should I edit in the docusaurus framework with TypeScript. Looking forward to your reply.
Best,
Kyrie

**Comment by leerob** [comment]
I think this would require creating a plugin: https://docusaurus.io/docs/using-plugins
